templates/taxi/car_detail.html

```diff
@@ -3,13 +3,18 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
 
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
+    {% if user in car.drivers.all %}
+      <form method="post" action="{% url 'taxi:car-remove-driver' pk=car.id %}">
+        {% csrf_token %}
+        <input type="submit" value="Delete me from this car" class="btn btn-danger link-to-page">
+      </form>
+    {% else %}
+      <form method="post" action="{% url 'taxi:car-add-driver' pk=car.id %}">
+        {% csrf_token %}
+        <input type="submit" value="Assign me to this car" class="btn btn-success link-to-page">
+      </form>
+    {% endif %}
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
```

Comment:
Ensure that the `car` object includes the `manufacturer` attribute with `name` and `country` fields. This is necessary for displaying the manufacturer's details correctly.